src/Og.php

```diff
@@ -419,4 +421,27 @@ public static function getSelectionHandler($entity, $bundle, $field_name, array
     return \Drupal::service('plugin.manager.entity_reference_selection')->createInstance('og:default', $options);
   }
 
+  /**
+   * Creates and saves an OG membership.
```

Comment:
> Currently it doesn't do the second one, should we add that in here?

If I remember correctly, if the membership already exists then we will have a mysql error, due to duplicate primary keys. I prefer to catch this error and throw a more descriptive error for a better DX.